Misc/Applications/RNAhybrid/makefile

```diff
@@ -33,6 +33,14 @@ subopt_%:
 		gapc_file="${*}.gap" \
 		gapc_binaryname="$@$(current_windowmodesuffix)";
 
+probing: probing_rnahybrid
+probing_%:
+	$(MAKE) compile \
+		gapc_product="alg_probing * alg_mfe * alg_pretty" \
```

Comment:
that is great, since it will return 3 results.
Could you try to make it an actual pareto product by replacing the first `*`  with a `^` and maybe use parentheses to explicitly indicate bindings, i.e. `(alg_probing ^ alg_mfe) * alg_pretty`

Reply:
I changed it to a pareto product.